model/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -43,6 +43,7 @@
         <class>org.keycloak.models.jpa.entities.IdentityProviderMapperEntity</class>
         <class>org.keycloak.models.jpa.entities.ProtocolMapperEntity</class>
         <class>org.keycloak.models.jpa.entities.UserConsentEntity</class>
+        <class>org.keycloak.grant.jpa.UserGrantEntity</class>
```

Comment:
By recent keycloak's convention, it might be better not to add or modify JPA entity because it causes a schema change. For example, Client Policies/Profiles avoid adding entities by storing JSON string itself to realm's attribute (existing key-value store).

Reply:
will be remove and stored as described above